### PR TITLE
Localised shift-left into hi

### PR DESCRIPTION
**Pull request by abhay-raj19**
### Describe your changes
Localised Shift left into hindi as suggested under #2061 

### Related issue number or link (ex: `resolves #issue-number`)
#2061 
### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Review by kumarankit999 — COMMENTED** [review]
Looking good here!
What about other Approvers! wait for them also!!

**Review by Garima-Negi — APPROVED** [review]
LGTM!

**Review by ankur0904 — APPROVED** [review]
LGTM :)

**Review by kumarankit999 — APPROVED** [review]
/lgtm

File: content/hi/shift-left.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+category: concept
+---
+
+## यह क्या है
+शिफ्ट लेफ्ट में "लेफ्ट" सॉफ़्टवेयर विकास जीवनचक्र के पहले चरणों की ओर संकेत करता है, जहां चरणों को बाएं से दाएं दिशा में निष्पादित किया जाता है। शिफ्ट लेफ्ट अभिक्रिया सॉफ़्टवेयर विकास जीवनचक्र के अंतिम चरणों की बजाय, प्रारंभिक चरणों में परीक्षण, सुरक्षा, या अन्य विकास प्रथाओं को संचालित करने की प्रथा है।
+
+प्रारंभ में परीक्षण की प्रक्रिया को वारंट किया जाता था, लेकिन अब शिफ्ट लेफ्ट को सुरक्षा और डिप्लॉयमेंट जैसे सॉफ़्टवेयर विकास और [डेवओप्स](/devops/), के अन्य पहलुओं में भी लागू किया जा सकता है।
+
+## समस्या
+सुरक्षा समस्याएं, बग और सॉफ़्टवेयर के दोषों को सुधारना और ठीक करना अधिक कठिन और महंगा हो सकता है, अगर उन्हें विकास चक्र के दौरान या डिप्लॉयमेंट के पश्चात में खोजा जाता है, विशेषतः अगर सॉफ़्टवेयर पहले से ही उत्पादन में डिप्लॉय हो चुका हो।
```
Comment (aj11anuj) [inline content/hi/shift-left.md:13]:
```suggestion
सुरक्षा समस्याएं, बग और सॉफ़्टवेयर के दोषों को सुधारना और ठीक करना अधिक कठिन और महंगा हो सकता है, अगर उन्हें विकास चक्र के दौरान या डिप्लॉयमेंट के पश्चात खोजा जाता है, विशेषतः अगर सॉफ़्टवेयर पहले से ही उत्पादन में डिप्लॉय हो चुका हो।
```

**Review by aj11anuj — CHANGES_REQUESTED** [review]

**Review by aj11anuj — APPROVED** [review]
lgtm

**Pull request merged**